src/lib/filter/adjudicators.ts

```diff
@@ -58,33 +58,60 @@ export const carriesDeletionTimestamp = pipe(
 );
 export const missingDeletionTimestamp = complement(carriesDeletionTimestamp);
 
-export const carriedKind = pipe(kubernetesObject => kubernetesObject?.metadata?.kind, defaultTo("not set"));
-export const carriedVersion = pipe(kubernetesObject => kubernetesObject?.metadata?.version, defaultTo("not set"));
-export const carriedName = pipe(kubernetesObject => kubernetesObject?.metadata?.name, defaultTo(""));
+export const carriedKind = pipe(
+  (kubernetesObject: KubernetesObject): string | undefined => kubernetesObject?.kind,
+  defaultTo("not set"),
+);
+export const carriedVersion = pipe(
+  (kubernetesObject: KubernetesObject): string | undefined => kubernetesObject?.metadata?.resourceVersion,
+  defaultTo("not set"),
+);
+export const carriedName = pipe(
+  (kubernetesObject: KubernetesObject): string | undefined => kubernetesObject?.metadata?.name,
+  defaultTo(""),
+);
 export const carriesName = pipe(carriedName, equals(""), not);
 export const missingName = complement(carriesName);
 
-export const carriedNamespace = pipe(kubernetesObject => kubernetesObject?.metadata?.namespace, defaultTo(""));
+export const carriedNamespace = pipe(
+  (kubernetesObject: KubernetesObject): string | undefined => kubernetesObject?.metadata?.namespace,
+  defaultTo(""),
+);
 export const carriesNamespace = pipe(carriedNamespace, equals(""), not);
 
-export const carriedAnnotations = pipe(kubernetesObject => kubernetesObject?.metadata?.annotations, defaultTo({}));
+export const carriedAnnotations = pipe(
+  (kubernetesObject: KubernetesObject): { [key: string]: string } | undefined =>
+    kubernetesObject?.metadata?.annotations,
+  defaultTo({}),
+);
 export const carriesAnnotations = pipe(carriedAnnotations, equals({}), not);
 
-export const carriedLabels = pipe(kubernetesObject => kubernetesObject?.metadata?.labels, defaultTo({}));
+export const carriedLabels = pipe(
+  (kubernetesObject: KubernetesObject): { [key: string]: string } | undefined => kubernetesObject?.metadata?.labels,
+  defaultTo({}),
+);
 export const carriesLabels = pipe(carriedLabels, equals({}), not);
 
 /*
   Binding collectors
 */
 
-export const definesDeletionTimestamp = pipe(binding => binding?.filters?.deletionTimestamp, defaultTo(false));
+export const definesDeletionTimestamp = pipe(
+  (binding: Binding): boolean => binding?.filters?.deletionTimestamp ?? false,
```

Comment:
Didn't we decide that `binding.filters.deletionTimestamp` would always exist now?  And be defaulted to `false`?    I'm trying to remember if we talked back around to requiring that `?? false` for some reason.